### DISCUSSION TITLE
How to pick the best threshold_match_probability through ROC Curve?

### DISCUSSION THREAD

**Discussion by yangg1224** [moj-analytical-services/splink#1855] [Q&A]
I am using ground truth table to plot the ROC curve, using 
`linker.roc_chart_from_labels_table("combined_GT")`

Here is the ROC curve:
![image](https://github.com/moj-analytical-services/splink/assets/50905062/63fa4308-d105-436f-aa22-95a3cc5161d5)
The screenshot show the top left corner point. 

```
linker = DuckDBLinker([AccessOAP,OACIS])
linker.load_model("model\saved_model_link.json")
results = linker.predict(threshold_match_probability=0.95)
```

Please help how to pick the best threshold_match_probability through the ROC curve...
 `results = linker.predict(threshold_match_probability= ? )
`

**Comment by RobinL** [comment]
There isn't really a single answer to this question I'm afraid - it really depends on your users preference for false positives vs false negatives.   In general, there's no reason to assume it would be the top left point (although it's often thereabouts)

**Reply by yangg1224** [reply]
Thanks for your reply, I wonder does the Match_Probability in the screenshot (0.0115%) is the value of threshold_match_probability, if I pick the top left corner point?
